notebooks/06-hmm.ipynb

```diff
@@ -247,12 +287,17 @@
    },
    "outputs": [],
    "source": [
-    "poor_hmm = pyemma.msm.estimate_hidden_markov_model(poor_dtraj, 2, lag=1)\n",
-    "good_hmm = pyemma.msm.estimate_hidden_markov_model(good_dtraj, 2, lag=1)\n",
+    "init_poor = dt.markov.hmm.init.discrete.metastable_from_data(poor_dtraj, n_hidden_states=2, lagtime=1)\n",
```

Comment:
We need to add some text here, that's a lot done in one cell (already done in ITS function above). I guess it requires to add some more info about the algorithmic details so people understand what it does.